Source/PeleLM.cpp

```diff
@@ -6643,13 +6603,14 @@ PeleLM::compute_scalar_advection_fluxes_and_divergence (const MultiFab& Force,
 
   MultiFab edgeflux[AMREX_SPACEDIM];
   MultiFab edgestate[AMREX_SPACEDIM];
+  int nghost = 2;
```

Comment:
Should this be 2 or some ngrow variable in the code +1 for EB?

Reply:
It must be 4, and it was defined before with the variable ng

Reply:
I meant, why is 4 the correct value?  Is it 2 + 1 + 1 (and if so, what are the different numbers for)?

Reply:
@emotheau Please add comments are variables that explain where the 4 comes from.